turbinia/config/turbinia_config_tmpl.py

```diff
@@ -103,59 +103,73 @@
 DEPENDENCIES = [{
     'job': 'BinaryExtractorJob',
     'programs': ['image_export.py'],
-    'docker_image': None
+    'docker_image': None,
+    'timeout': 300
 }, {
     'job': 'BulkExtractorJob',
     'programs': ['bulk_extractor'],
-    'docker_image': None
+    'docker_image': None,
+    'timeout': 300
 }, {
     'job': 'FsstatJob',
     'programs': ['fsstat'],
-    'docker_image': None
+    'docker_image': None,
+    'timeout': 300
 }, {
     'job': 'GrepJob',
     'programs': ['grep'],
-    'docker_image': None
+    'docker_image': None,
+    'timeout': 300
 }, {
     'job': 'HadoopAnalysisJob',
     'programs': ['strings'],
-    'docker_image': None
+    'docker_image': None,
+    'timeout': 300
 }, {
     'job': 'HindsightJob',
     'programs': ['hindsight.py'],
-    'docker_image': None
+    'docker_image': None,
+    'timeout': 300
 }, {
     'job': 'JenkinsAnalysisJob',
     'programs': ['john'],
-    'docker_image': None
+    'docker_image': None,
+    'timeout': 300
 }, {
     'job': 'PartitionEnumerationJob',
     'programs': ['bdemount'],
-    'docker_image': None
+    'docker_image': None,
+    'timeout': 300
 }, {
     'job': 'PlasoJob',
     'programs': ['log2timeline.py'],
-    'docker_image': None
+    'docker_image': None,
+    'timeout': '3600'
 }, {
     'job': 'PhotorecJob',
     'programs': ['photorec'],
-    'docker_image': None
+    'docker_image': None,
+    'timeout': 300
```

Comment:
Anything that has to parse every block on the disk should probably have a higher timeout (including strings below).  Maybe an hour?